ES/ESmy006.xml

```diff
@@ -189,128 +592,248 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              prefatory texts. In some parts of the manuscript the scribe reduced the size of the script.</desc>
+                                                 </handNote>
+
                   </handDesc>
                   <decoDesc>
 
-                     <decoNote type="frame" xml:id="d1">
-                        <locus target="#106ra">106ra</locus>: fine, narrow ornamental band;
+                     <decoNote type="band" xml:id="d1">
+                        <locus target="#106ra"/>Fine colored ornamental band;
                         geometric motifs.</decoNote>
                   </decoDesc>
                   <additions>
                      <list>
                         <item xml:id="a1">
-                           <locus target="#73rb">f. 73rb-73</locus>
-                           <desc>va: Donation note.</desc>
-                           <quote xml:lang="gez">
-                              <locus target="#73rb">73rb</locus>: ዝመጽሐፍ፡ ፬ወንጌል፡ ዘራስ፡ ስብሐት፡ ዘወሐቦ፡
-                              ለቅዱስ፡ ዮሐንስ፡ ደብረ፡ ማዕፆ፡ ከመ፡ ይኵኖ፡ መርሐ፡ በመንግስተ፡ ሰማያት፡ ወስመ፡ ክርስትናሁኒ፡ ኪዳነ፡
-                              ማርያም፡ ወስመ፡ እሙ፡ ወለተ፡ ስላሴ፡ ወስመ፡ ብእሲቱ፡ ወለተ፡ ማርያም፡ ወስመ፡ ደቂቁ፡ ገብረ፡ ሕይወት፡
-                              ወልደ፡ ገብርኤል፡ አርአያ፡ ሥላሴ፡ ኦአምላከ፡ ዮሐንስ፡ ዕቀብ፡ ሕይወቶ፡ ወስረይ፡ ኀጢአቶ፡
-                                 ወአግርር[<locus target="#73va">73va</locus>]፡ ፀሮ፡ ታህተ፡ እገሪሁ፡ ለገብርከ፡
+                           <locus from="73rb" to="73va"/>
+                           <desc type="DonationNote"/>
+                           <quote xml:lang="gez">ዝመጽሐፍ፡ ፬ወንጌል፡
+                             ዘ<persName ref="PRS8668SebhatA" role="donor">ራስ፡ ስብሐት፡</persName> ዘወሐቦ፡ ለ<placeName ref="INS0039MY">ቅዱስ፡ ዮሐንስ፡ ደብረ፡ ማዕፆ፡</placeName>
+                             ከመ፡ ይኵኖ፡ መርሐ፡ በመንግስተ፡ ሰማያት፡ ወስመ፡ ክርስትናሁኒ፡ ኪዳነ፡
+                              ማርያም፡ ወስመ፡ እሙ፡ <persName ref="PRS13295WalattaSellase">ወለተ፡ ስላሴ፡</persName> ወስመ፡ ብእሲቱ፡ <persName ref="PRS13296WalattaMaryam">
+                            ወለተ፡ ማርያም፡</persName> ወስመ፡ ደቂቁ፡
+                              <persName ref="PRS13291GabraHeywat">ገብረ፡ ሕይወት፡</persName>
+                              <persName ref="PRS13292WaldaGabreel">ወልደ፡ ገብርኤል፡</persName> <persName ref="PRS13293ArayaSellase">
+                            አርአያ፡ ሥላሴ፡</persName> ኦአምላከ፡ ዮሐንስ፡ ዕቀብ፡ ሕይወቶ፡ ወስረይ፡ ኀጢአቶ፡
+                                ወአግርር <pb n="73va"/>፡ ፀሮ፡ ታህተ፡ እገሪሁ፡ ለገብርከ፡
                               ኪዳነ፡ ማርያም፡ ወለኵሎሙ፡ ሰብአ፡ ቤቱ፡ ይዕቀቦሙ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ለዝንቱኒ፡ መጽሐፋ፡ ወንጌል፡
-                              ዘሠረቆ፡ ወዘፈሐቆ፡ በስልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ውጉዘ፡ ለይኩን፨ ዝ፬ቱ፡ ወንጌል፡ ዘራስ፡ ስብሐት፡
-                              ዘተሣየጦ፡ በንዋዩ፨ </quote>
+                              ዘሠረቆ፡ ወዘፈሐቆ፡ በስልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ውጉዘ፡ ለይኩን፨ ዝ፬ቱ፡ ወንጌል፡
+                              ዘ<persName ref="PRS8668SebhatA">ራስ፡ ስብሐት፡</persName>
+                              ዘተሣየጦ፡ በንዋዩ፨</quote>
                            <quote xml:lang="en">
```

Comment:
`<quote xml:lang="en">` should become `<foreign xml:lang="en">`, nested inside the main "`q`" element (as it is not a separate quote but translation of the quote above)

Reply:
done